docs/configuration/config-file-reference.md

```diff
@@ -3969,3 +3972,35 @@ The `s3_sse_config` configures the S3 server-side encryption. The supported CLI
 # CLI flag: -<prefix>.s3.sse.kms-encryption-context
 [kms_encryption_context: <string> | default = ""]
 ```
+
+### `tracing_config`
+
+The `tracing_config` configures backends cortex uses.
+
+```yaml
+# Set to false to disable tracing.
+# CLI flag: -tracing.enabled
+[enabled: <boolean> | default = true]
+
+# Tracing type. OTEL and JAEGER are currently supported
+# CLI flag: -tracing.type
+[type: <string> | default = "jaeger"]
+
+otel:
+  # otl collector endpoint that the driver will use to send spans
+  # CLI flag: -tracing.otel.oltp-endpoint
+  [oltp_endpoint: <string> | default = ""]
+
+  # Disables client transport security for the exporter.
+  # CLI flag: -tracing.otel.insecure
+  [insecure: <boolean> | default = false]
+
+  # enhance exporter with attributes for specific exporter. Currently only
+  # awsxray is supported
+  # CLI flag: -tracing.otel.exporter-type
+  [exporter_type: <string> | default = ""]
```

Comment:
does default "" mean awsxray?

Reply:
No.. defaults mean "genetic"...

What we do here  for some destinations we need to configure the exporter a little different. For instance, if the end backend is X-Ray we need to generate the traceIds in a specific format. 

I believe for the majority of the backends it will work with this type set to empty.